repos/system_upgrade/common/actors/removeupgradefailedflag/actor.py

```diff
@@ -0,0 +1,21 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import removeupgradefailedflag
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class RemoveUpgradeFailedFlag(Actor):
+    """
+    Removes upgrade failed flag file
+
+    The upgrade failed flag file is set in the initramfs when the upgrade fails
```

Comment:
```
In case the in-place upgrade fails in the initramdisk "phase", the Leapp process creates the "/var/log/leapp/.leapp_upgrade_failed" flag file to prevent re-running this part from scratch. We need to remove the flag file before the next run.
```